lib/bootstrap/bundler.rb

```diff
@@ -56,10 +56,19 @@ def setup!(options = {})
     # execute bundle install and capture any $stdout output. any raised exception in the process will be trapped
     # and returned. logs errors to $stdout.
     # @param options [Hash] invoke options with default values, :max_tries => 10, :clean => false, :install => false, :update => false
-    # @param   options[:update] must be either false or a String or an Array of String
+    # @param options[:update] must be either false or a String or an Array of String
+    # Available options are:
```

Comment:
you can use the correct `@option` yard tag here see http://www.rubydoc.info/gems/yard/file/docs/Tags.md#option